config/rhoai/manager_metrics_patch.yaml

```diff
@@ -9,6 +9,9 @@ spec:
       containers:
       - name: manager
         ports:
-        - containerPort: 8443
+        - containerPort: 8080
           protocol: TCP
           name: metrics
+        - containerPort: 8443
```

Comment:
i assume, https://github.com/opendatahub-io/opendatahub-operator/blame/rhoai/config/monitoring/prometheus/apps/prometheus-configs.yaml#L336 need to be updated as well?

Reply:
cc @akram

Reply:
@zdtsw nope, that's the port number a few lines after that has to be changed to 8443 and it has been done already.
The /metrics path remains the same here.